services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+import datetime
+
+from django.db.models.query import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime.datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie_session = MovieSession.objects.create(show_time=movie_show_time,
+                                                movie_id=movie_id,
+                                                cinema_hall_id=cinema_hall_id)
+    return movie_session
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+
+    if session_date:
+        movie_session = MovieSession.objects.filter(
+            show_time__date=session_date
+        )
+
+    return movie_session
+
+
+def get_movie_session_by_id(movie_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: int = None,
+                         cinema_hall_id: int = None) -> MovieSession:
+    movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
try to rewrite this logic for efficiency, so you don't have to get session and save it in case if you didn't get any optional params. Also returning movie_session is not a requirement